operator/cmd/mesh/install.go

```diff
@@ -175,6 +185,18 @@ func Install(rootArgs *RootArgs, iArgs *InstallArgs, logOpts *log.Options, stdOu
 	// "no running Istio pods in istio-system" for the first time
 	_ = detectIstioVersionDiff(p, tag, ns, kubeClient, setFlags)
 
+	if iArgs.ShowDiff {
+		diff, err := compareIOPWithInstalledIOP(kubeClient, iop)
+		if err != nil && !errors.IsNotFound(err) {
```

Comment:
You don't need the `err != nil` check here because the `IsNotFound` returns false if it's nil.